content/academy/professional_scraping/solutions/actor_building.md

```diff
@@ -513,6 +500,46 @@ We've also replaced the default `title` and `description` with our own custom on
 
 Let's try it out now! Input **iphone** into the box labeled **keyword**, click **Start**, and see the sparks fly!
 
+## [](#quiz-answers) Quiz answers 📝
+
+- When using Puppeteer or Playwright, how can you still use jQuery with the SDK?
+
+There are two ways. You can either use the [injectJQuery](https://sdk.apify.com/docs/api/puppeteer#puppeteerinjectjquerypage) utility function which will enable you to use jQuery inside of `page.evalute()`, or you can use Cheerio to load the page's content like this:
+
+```JavaScript
+const $ = cheerio.load(await page.content());
+```
+
+Which will allow you to use jQuery syntax outside of Puppeteer/Playwright page evaluation functions.
+
+- What is the main difference between Cheerio and jQuery?
+
+Though they share the same syntax, and therefore look quite similar, Cheerio and jQuery are designed for different purposes. jQuery is a frontend library for manipulating the DOM, making AJAX calls, etc. while Cheerio is a backend library exclusively for Node.js built for the purpose of intuitively parsing HTML on the server-side.
```

Comment:
I would add something like: `Some specific functionality of JQuery is not available with Cheerio, follow official documentation if you run into one of these cases`